web-services-app/src/main/java/edu/unc/lib/boxc/web/services/processing/IiifV3ManifestService.java

```diff
@@ -63,7 +64,7 @@ public class IiifV3ManifestService {
     public static final String WIDTH = "width";
     public static final String HEIGHT = "height";
     private static final List<String> FILE_TYPES = Arrays.asList(VIDEO_MP4.toString(), VIDEO_MPEG.toString(),
-            VIDEO_QUICKTIME.toString(), AUDIO_MP4.toString(), AUDIO_MPEG.toString());
+            VIDEO_QUICKTIME.toString(), AUDIO_MP4.toString(), AUDIO_MPEG.toString(), AUDIO_AAC.toString());
```

Comment:
rather than expanding this, I think we should update `IIIFv3ViewableFilter` to add another datastream filter, so that it'll find records that have the audio access datastream (in addition to jp2s and stuff matching the mimetype)